frontend/taipy-gui/package.json

```diff
@@ -50,7 +51,8 @@
     "types-base": "dts-bundle-generator -o base/src/packaging/taipy-gui-base.gen.d.ts base/src/exports.ts",
     "doc": "typedoc --plugin typedoc-plugin-markdown --excludeNotDocumented --disableSources src/extensions/exports.ts",
     "doc.json": "typedoc --json docs/taipy-gui.json src/extensions/exports.ts",
-    "mkdocs": "typedoc --options typedoc-mkdocs.json"
+    "mkdocs": "typedoc --options typedoc-mkdocs.json",
+    "pack-base": "cd ../../taipy/gui/webapp/taipy-gui-base-export && npm pack && mv taipy-gui-base-4.1.0.tgz ../taipy-gui-base.tgz && cd ../ && rm -rf taipy-gui-base-export"
```

Comment:
does this work on windows ?

Reply:
I will try it out before these are merged into develop, which will not be the case for the upcoming weeks